app/views/components/search.scala.html

```diff
@@ -17,28 +17,25 @@
 @import play.twirl.api.Html
 
 @this(
-    paragraph: components.paragraph,
-    subHeading: components.sub_heading,
-    input: components.input_text,
-    button: components.button,
-    formWithCSRF: uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF
-    )
+        paragraph: components.paragraph,
+        subHeading: components.sub_heading,
+        input: components.input_text,
+        button: components.button,
+        formWithCSRF: uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF
+)
 
 @(form: Form[String], postAction: Call)(implicit request: IdentifierRequest[_], messages: Messages)
 
-@key = @{"search"}
+    @key = @{"search"}
 
-<br/>
-@subHeading(Html(messages(s"$key.title")), classes = "govuk-heading-m")
-@paragraph(Html(messages(s"$key.desc")))
+    <br/>
+    @subHeading(Html(messages(s"$key.title")), classes = "govuk-heading-m")
+    @paragraph(Html(messages(s"$key.desc")))
 
-@formWithCSRF(postAction, 'novalidate -> "novalidate", 'style -> "display: flex; gap: 5px; flex-direction: row;", 'class -> "govuk-!-width-two-thirds") {
-    @input(
```

Comment:
The problem with this @input object is that it creates a wrapper div with class "govuk-form-group" which wraps the input only, whereas we need that div to wrap both the input and submit button. 
This is why we were having difficulty showing the button on the same line.

Reply:
We have created a ticket for this for Tony